jax/_src/lax/lax.py

```diff
@@ -801,24 +803,114 @@ def abs(x: ArrayLike) -> Array:
   """
   return abs_p.bind(x)
 
+@export
 def pow(x: ArrayLike, y: ArrayLike) -> Array:
-  r"""Elementwise power: :math:`x^y`."""
+  r"""Elementwise power: :math:`x^y`.
+
+  This function lowers directly to the `stablehlo.pow`_ operation, along with
+  a `stablehlo.convert`_ when the argument dtypes do not match.
+
+  Args:
+    x: Input array giving the base value. Must have floating or complex type.
+    y: Input array giving the exponent value. Must have integer, floating, or
+      complex type. Its dtype will be cast to that of ``x.dtype`` if necessary.
+      If neither ``x`` nor ``y`` is a scalar, then ``x`` and ``y`` must have
+      the same number of dimensions and be broadcast-compatible.
+
+  Returns:
+    An array of the same dtype as ``x`` containing the elementwise power.
+
+  See also:
+    :func:`jax.lax.integer_pow`: Elementwise power where ``y`` is a fixed integer.
```

Comment:
Should we use "static" instead of "fixed"?

Reply:
Good idea - done!